runtime/interpreter/value.go

```diff
@@ -15260,6 +15421,20 @@ type PathValue struct {
 	Identifier string
 }
 
+func NewUnmeteredPathValue(domain common.PathDomain, identifier string) PathValue {
+	return PathValue{domain, identifier}
+}
+
+func NewPathValue(memoryGauge common.MemoryGauge, domain common.PathDomain, identifier string) PathValue {
+	if memoryGauge != nil {
+		memoryGauge.UseMemory(common.MemoryUsage{
+			Kind:   common.MemoryKindPathValue,
+			Amount: uint64(len(identifier)),
```

Comment:
I use two approaches to length:
1. Make sure it's at least zero.
2. Assume it's non-negative.

Is the second approach safe or is the first necessary?